### PR TITLE
Fix FACTERLIB examples, update style, and fix typos in Facter 1.x/2.x docs

### PR DESCRIPTION
**Pull request by gguillotte**
Resolve the discrepancy pointed out in [FACT-1234](https://tickets.puppetlabs.com/browse/FACT-1234).

Also, fix several typos and apply style and formatting consistently across the Facter 1.x and 2.x docs.

(Attn @kylog @peterhuene @nfagerlund)

### CODE REVIEW

File: source/facter/1.6/custom_facts.markdown [hunk 1]
```diff
@@ -18,32 +11,32 @@ Sometimes you need to be able to write conditional expressions
...
 to distribute the facts to the client.
 
 ## Loading Custom Facts
 
 Facter offers a few methods of loading facts:
 
- * $LOAD\_PATH, or the ruby library load path
- * The environment variable 'FACTERLIB'
- * Facts distributed using pluginsync
+* `$LOAD\_PATH`, or the Ruby library load path
```
Comment (nfagerlund) [inline source/facter/1.6/custom_facts.markdown]: If you wrap it in backticks, you don't need the backslash escape on the underscore. (Actually, on our MD renderer this never needed it.)

**Comment by nfagerlund** [comment]
Other than the backslash-underscore thing (can probably just do a multi-file find-and-replace for `\_`), :+1:

**Comment by kylog** [comment]
:+1:

**Comment by gguillotte** [comment]
Done! Slamming the green button.

**Pull request merged**